manifest.json

```diff
@@ -540,14 +540,14 @@
         }
       },
       "waveSurferJs": {
-        "version": "1.0.46",
-        "downloadFormat": "files",
-        "url": "https://cdn.rawgit.com/katspaugh/wavesurfer.js/v1.0.46/dist/",
-        "rootDirPrefix": "waveSurfer-js-",
+        "version": "1.4.0",
+        "downloadFormat": "zip",
+        "url": "https://github.com/katspaugh/wavesurfer.js/archive/1.4.0.zip",
```

Comment:
1.4.0 is the latest version?

Reply:
btw, if we need just one file do we have to download the entire repo? (isn't there any way to download the dist/ files only?)

Reply:
I found a cdn here: https://unpkg.com/wavesurfer.js@2.2.1/dist/wavesurfer.min.js (You can change the version no. as well) will it be useful?

Reply:
Yes this builds as expected with other libs in third_party/generated hence this file can be used.
I'll test this and update you asap Thanks

Reply:
@NishealJ, I'm not sure whether [the v2+ will work with our setup](https://wavesurfer-js.org/v2/), if it doesn't work yuo can use 1.4.0:  https://unpkg.com/wavesurfer.js@1.4.0/dist/wavesurfer.min.js

Reply:
@DubeySandeep I've tried using v2+ as zip previously but failed because v2+ has completely modular code which is difficult to combine with our third_party.js as they were having a call to modules which were not available in /generated.

but the CDN seems working good as it is minified and made for distribution so has no modular code [here](https://drive.google.com/open?id=12TlzOx-XHeBFH89WhXkNY0nAwKojS1Tq) is quick video of v2+ PTAL Thanks!